docs/.vuepress/public/_redirects

```diff
@@ -67,4 +67,4 @@
 /docs/quick-start/synology.html             /docs/guides/synology.html
 
 /docs/client.html                           /docs/tcp/client.html
-/docs/topics/tcp-support.html               /docs/tcp/
\ No newline at end of file
+/docs/topics/tcp-support.html               /docs/tcp/
```

Comment:
I think we need to add additional redirects
`docs/docs/install/helm.md → docs/docs/k8s/helm.md`

Reply:
Good catch, added in 9f7c2107b2c306b15598b0124b92a4e59e5f23b5

Reply:
https://github.com/pomerium/pomerium/pull/2667/files#diff-3075853952d74de2c8c5c1bc8f46f318e52f5b0d3e1afd68c949e9519983b90cR72

This redirect file is handled by Netlify, and uses the web URL instead of the filename.